packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -125,24 +162,25 @@ export class EthereumAnchorService implements AnchorService {
    * @param params - a RequestAnchorParams object
    * @private
    */
-  private _makeAnchorRequest(params: RequestAnchorParams): Observable<AnchorServiceResponse> {
+  private _makeAnchorRequest(
+    streamID: StreamID,
+    tip: CID,
+    carFile: CAR
+  ): Observable<AnchorServiceResponse> {
     return defer(() =>
       from(
         this.sendRequest(this.requestsApiEndpoint, {
           method: 'POST',
```

Comment:
How do we distinguish new vs old requests on CAS side?
Can this be done based on `Content-Type` perhaps?

Reply:
Yes, we'll be distinguishing by `Content-Type`.